src/node.cc

```diff
@@ -3261,6 +3258,9 @@ void LoadEnvironment(Environment* env) {
   // are not safe to ignore.
   try_catch.SetVerbose(false);
 
+  // Compile, execute the lib/internal/bootstrap_node.js file. (Which was
```

Comment:
`Compile and execute`. I know it was like this before, but if we're moving it, we might as well fix it.